tests/unittests/utils/test_transformer_pooler.py

```
@@ -0,0 +1,95 @@
+from unittest import mock
+
+import numpy as np
+import pytest
+import torch
+
+
+def _make_mixin(get_text_features_return):
+    """Build a minimal ZeroShotTransformerPromptMixin instance with mocks."""
+    try:
+        from fiftyone.utils.transformers import ZeroShotTransformerPromptMixin
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)